espn_api/basketball/player.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, data, year, pro_team_schedule = None):
         self.posRank = json_parsing(data, 'positionalRanking')
         self.stats = {}
         self.schedule = {}
+        expected_return_date = json_parsing(data, 'expectedReturnDate')
+        self.expectedReturnDate = datetime(*expected_return_date).date() if expected_return_date else None
```

Comment:
We can set this var name to `expected_return_date`, to follow better python standard. It was my bad habit of following the API name vars in the beginning of creating this package.